packages/@uppy/companion/src/server/helpers/request.js

```diff
@@ -131,11 +134,15 @@ exports.getURLMeta = async (url, blockLocalIPs = false) => {
         .on('response', (response) => {
           // Can be undefined for unknown length URLs, e.g. transfer-encoding: chunked
           const contentLength = parseInt(response.headers['content-length'], 10)
+          const filename = response.headers['content-disposition']
+            ? contentDisposition.parse(response.headers['content-disposition']).parameters.filename
+            : `${path.basename(response.request.requestUrl)}-${randomUUID()}`
```

Comment:
added comment

Reply:
can we document this better? I think it's obvious that we will avoid duplicate names by adding a random uuid, but why do we need to avoid duplicate file names? if someone uploads the same file twice, wouldn't they want the same name? Also I can imagine many people wouldn't want a random string appended to their file names, so I'm thinking we will soon get a request for an option to turn this off?